ZFMAGFLD/iocBoot/iocZFMAGFLD-IOC-01/config.xml

```diff
@@ -26,6 +26,8 @@
         <macro name="X_DAQ_CHANNEL" pattern="^ai[0-9]+$" description="DAQ channel to read for the X axis" defaultValue="ai0" hasDefault="YES" />
         <macro name="Y_DAQ_CHANNEL" pattern="^ai[0-9]+$" description="DAQ channel to read for the Y axis" defaultValue="ai1" hasDefault="YES" />
         <macro name="Z_DAQ_CHANNEL" pattern="^ai[0-9]+$" description="DAQ channel to read for the Z axis" defaultValue="ai2" hasDefault="YES" />
+        
+        <macro name="EXTRA_DAQ_CHANNEL" pattern="^.*$" description="Expose an additional DAQ channel. Leave empty to disable." defaultValue="" hasDefault="YES" />
```

Comment:
Could we instead make this more like the other channels, where you specify the channel itself but if you specify nothing then it doesn't load it?

Reply:
Also, nothing about the name of macro or the description makes it clear this has anything to do with the magnet rotation